src/components/SafeListSidebar/SafeList/index.tsx

```diff
@@ -74,7 +74,7 @@ const isSameAddress = (addrA: string, addrB: string): boolean => addrA.toLowerCa
 export const SafeList = ({ onSafeClick }: Props): ReactElement => {
   const classes = useStyles()
   const currentSafeAddress = extractSafeAddress()
-  const ownedSafes = useOwnerSafes()
+  const { ownerSafes } = useOwnerSafes()
```

Comment:
Perhaps we can use the new `isLoading` flag here somehow as well? A spinner?

Reply:
Good idea, if we decide to display the Safe List Sidebar by default a loading state would be useful!